.travis.yml

```diff
@@ -14,7 +14,7 @@ cache:
   directories:
     - node_modules
 node_js:
-  - "4.2"
+  - "6.3"
```

Comment:
I saw you are using default function parameters, it needs node > 6.8, see: http://node.green/